sprotty-elk/package.json

```diff
@@ -27,22 +27,26 @@
     "url": "https://github.com/eclipse/sprotty-layout.git"
   },
   "dependencies": {
+    "elkjs": "^0.4.1",
     "sprotty": "^0.4.2"
   },
   "devDependencies": {
+    "@types/chai": "^4.1.7",
+    "@types/mocha": "^5.2.5",
     "chai": "^4.2.0",
     "jenkins-mocha": "^6.0.0",
     "mocha": "^5.2.0",
+    "reflect-metadata": "^0.1.12",
     "rimraf": "^2.6.2",
-    "ts-node": "^7.0.1",
+    "ts-node": "<7.0.0",
```

Comment:
Is that constraint still necessary?

Reply:
Yes, mocha tests did not work with ts-node 7.0.1.